dagmc_stats/dagmc_stats.py

```diff
@@ -471,12 +474,88 @@ def get_roughness(my_core, native_ranges):
 
     outputs
     -------
-    roughness : (numpy array) the roughness for all surfaces in the meshset
+    roughness : (dictionary) the roughness for all vertices in the meshset
+    stored in the form of vert : local roughness value
     """
     tri_vert_data, all_verts = get_tri_vert_data(my_core,
                                                     native_ranges[types.MBTRI])
     gc_all = get_gaussian_curvature(my_core, all_verts, tri_vert_data)
-    roughness = np.zeros(len(all_verts))
-    for vert_idx, vert_i in enumerate(all_verts):
-        roughness[vert_idx] = get_lri(vert_i, gc_all, tri_vert_data, my_core)
+    roughness = {}
+    for vert_i in all_verts:
+        roughness[vert_i] = get_lri(vert_i, gc_all, tri_vert_data, my_core)
     return roughness
+    
+
+def get_tri_roughness(my_core, native_ranges, roughness):
```

Comment:
Might be useful if this allowed the user to pass in a list of triangles, like `get_area_triangle()` above.

Reply:
Actually - let's wait on this until we move to Pandas, because at some point we probably want to just populate this for the whole set of vertices